client/api/python/heketi/test_client.py

```diff
@@ -0,0 +1,287 @@
+#
+# Copyright (c) 2015 The heketi Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import unittest
+from heketi import *
+
```

Comment:
Sweet.  Does this test require heketi to be running?  If it does, it should probably document the setup needed.